contracts/ownership/Secondary.sol

```diff
@@ -22,13 +27,20 @@ contract Secondary {
     _;
   }
 
+  /**
+   * @return the address of the primary.
+   */
   function primary() public view returns (address) {
     return _primary;
   }
-
+  
+  /**
+   * @dev Transfers primariness to a recipient.
```

Comment:
I think primariness is a terrible word X-)
Maybe: Transfers the contract to a new primary.

Reply:
updated it. Thanks!